docs/BasicCommunication/OnEventChanged/index.md

```diff
@@ -18,20 +18,18 @@ SDL uses `OnEventChanged` notification to appropriately manage the hmiLevel and
 !!! MUST
 1. Send notification with appropriate parameter value when active call on HMI has been started/ended.
 
-2. Send [SDL.OnAppDeactivated](../../sdl/onappdeactivated) to the active app when the phone call is started.
-
-3. Resume the applications to their original state prior to the phone call event in the HMI when the event ends (see note below).
+2. Resume the applications to their original audio streaming state prior to the phone call event in the HMI when the event ends (see note below).
```

Comment:
```suggestion
2. Return the applications to their appropriate audio streaming state in the HMI when the event ends.
```
Realizing this is slightly misleading, since a different app could have been activated during the event, meaning that we shouldn't resume the state if it's no longer applicable.